pages/collections.py

```diff
@@ -84,3 +85,96 @@ class CollectionSubmitPage(BaseCollectionPage):
         By.CSS_SELECTOR,
         '[data-test-collection-submission-confirmation-modal-add-button]',
     )
+
+
+class CollectionEditPage(BaseCollectionPage):
+    url_addition = '{guid}/edit'
+
+    identity = Locator(By.CSS_SELECTOR, 'div[data-test-submit-section-click-to-edit]')
+    remove_button = Locator(
+        By.CSS_SELECTOR, 'span[data-test-collections-remove-button] > button'
+    )
+    modal_remove_reason_input = Locator(
+        By.CSS_SELECTOR, 'textarea[data-test-collections-remove-reason]'
+    )
+    modal_cancel_remove_button = Locator(
+        By.CSS_SELECTOR, 'button[data-test-cancel-delete]'
+    )
+    modal_remove_button = Locator(By.CSS_SELECTOR, 'button[data-test-confirm-delete]')
+
+    def __init__(self, driver, verify=False, provider=None, guid=''):
+        self.guid = guid
+        super().__init__(driver, verify, provider)
+
+    @property
+    def url(self):
+        return (
+            urljoin(self.base_url, self.provider_id)
+            + '/'
+            + self.url_addition.format(guid=self.guid)
+        )
+
+
+class BaseCollectionModerationPage(BaseCollectionPage):
+    loading_indicator = Locator(By.CSS_SELECTOR, '.ball-scale')
+
+    submission_cards = GroupLocator(By.CSS_SELECTOR, '[data-test-submission-card]')
+
+    def get_submission_card(self, node_id):
+        for card in self.submission_cards:
+            url = card.find_element_by_css_selector(
+                '[data-test-submission-card-title]'
+            ).get_attribute('href')
+            guid = url.split(settings.OSF_HOME + '/', 1)[1]
+            if guid == node_id:
+                return card
+            return None
```

Comment:
As discussed, missed indentation for `return`.

In fact, we can remove the `return` since `return None` is default in Python.

Reply:
Fixed.